tempodb/encoding/vparquet3/block_traceql_test.go

```diff
@@ -498,13 +498,17 @@ func fullyPopulatedTestTrace(id common.ID) *Trace {
 									{Key: "unsupported-kv-list", ValueDropped: kvListValue, ValueType: attrTypeNotSupported},
 								},
 								Events: []Event{
-									{TimeUnixNano: 1, Name: "e1", Attrs: []EventAttribute{
-										{Key: "foo", Value: []byte("fake proto encoded data. i hope this never matters")},
-										{Key: "bar", Value: []byte("fake proto encoded data. i hope this never matters")},
-									}},
-									{TimeUnixNano: 2, Name: "e2", Attrs: []EventAttribute{}},
+									{
+										TimeSinceStartUnixNano: 1,
+										Name:                   "e1",
+										Attrs: []Attribute{
+											attr("foo", "fake proto encoded data. i hope this never matters"),
+											attr("bar", "fake proto encoded data. i hope this never matters"),
+										},
+									},
+									{TimeSinceStartUnixNano: 2, Name: "e2", Attrs: []Attribute{}},
 								},
-								Links: linkBytes,
+								Links: []Link{},
```

Comment:
* `linkBytes` contained an encoded link, it would be good to define similar link data here
* `linkBytes` and `links` defined in line 393-414 is no longer needed and can be removed